src/codemagic/tools/app_store_connect/action_groups/certificates_action_group.py

```diff
@@ -0,0 +1,228 @@
+from __future__ import annotations
+
+import pathlib
+from abc import ABCMeta
+from typing import List
+from typing import Optional
+from typing import Sequence
+from typing import Set
+from typing import Union
+from typing import cast
+
+from codemagic import cli
+from codemagic.apple.resources import CertificateType
+from codemagic.apple.resources import ProfileType
+from codemagic.apple.resources import ResourceId
+from codemagic.apple.resources import SigningCertificate
+from codemagic.cli import Colors
+from codemagic.models import Certificate
+from codemagic.models import PrivateKey
+
+from ..abstract_base_action import AbstractBaseAction
+from ..action_group import AppStoreConnectActionGroup
+from ..arguments import CertificateArgument
+from ..arguments import CommonArgument
+from ..arguments import Types
+from ..errors import AppStoreConnectError
+
+
+class CertificatesActionGroup(AbstractBaseAction, metaclass=ABCMeta):
+    @cli.action(
+        "create",
+        CertificateArgument.CERTIFICATE_TYPE,
+        CertificateArgument.PRIVATE_KEY,
+        CertificateArgument.PRIVATE_KEY_PASSWORD,
+        CertificateArgument.P12_CONTAINER_PASSWORD,
+        CertificateArgument.P12_CONTAINER_SAVE_PATH,
+        CommonArgument.SAVE,
+        action_group=AppStoreConnectActionGroup.CERTIFICATES,
+        deprecation_info=cli.ActionDeprecationInfo("create-certificate", "0.49.0"),
+    )
+    def create_certificate(
```

Comment:
Previously defined in https://github.com/codemagic-ci-cd/cli-tools/blob/3670fa624997f17cb437bc88198f7bdee3594f35/src/codemagic/tools/app_store_connect.py#L474-L509